src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -702,8 +705,9 @@ export default class CanvasController {
             return;
         }
 
-        const textAtPoint = [];
-        const { sidLines } = NavigationLibrary;
+        const textAtFix = [];
+        const sidLines = NavigationLibrary.getProcedureLines(PROCEDURE_TYPE.SID);
+
         let mostRecentFixName = '';
```

Comment:
This var is not used, and for me shows a lint error to that effect. It should be deleted if not needed.

Reply:
done